analyses/immune-deconv/02-visualize_quantiseq.Rmd

```diff
@@ -221,10 +231,11 @@ ggsave(cg_celltypes_plot_file, cg_plot, width = 11, height = 3.5)
 # First, remove cd8_cd4_ratio
 quantiseq_no_ratio <- quantiseq_subset %>%
   filter(cell_type != "cd8_cd4_ratio")
+
 subtypes_celltypes_plot <- ggplot(quantiseq_no_ratio) +
-  aes(x = molecular_subtype, y = score, color = cancer_group_hex) + 
+  aes(x = molecular_subtype, y = score) + 
   geom_boxplot(outlier.shape = NA, color = "grey40", size = 0.2) + 
-  geom_jitter(width = 0.15, size = 0.75, alpha = 0.6) + 
+  geom_jitter(shape = 1, width = 0.15, size = 0.75, alpha = 0.4, aes(color = cancer_group_hex)) + 
```

Comment:
By having `shape=1` here but not also in the legend, we have a mismatch. I'd recommend removing `shape=1` from here (or you have to add `shape=1` to the `geom_point()` making the legend).